### DISCUSSION TITLE
How to get the proportion of cells accessible for a certain peak

### DISCUSSION THREAD

**Discussion by siyuyang1013** [GreenleafLab/ArchR#2038] [Questions / Documentation]
Hello~
How to get the proportion of cells accessible for a certain peak, for example, chr1:800-1300 This peak has 100 cells accessible in the microglia, and the total number of cells in the microglia is 1000, then the proportion of accessible open for this peak is 100/1000

**Comment by Zepeng-Mu** [comment]
Isn't it the same as getting the peakMatrix and count how many cells have non-zero values for that given peak?

**Reply by siyuyang1013** [reply]
But when I extracted the peak matrix, I realized that this matrix has no row names, i.e. no peak names
> peaks.mat
class: RangedSummarizedExperiment
dim: 447926 33409
metadata(0):
assays(1): PeakMatrix
rownames: NULL
rowData names(1): idx
colnames(33409): F#AAAGATGCAGGTCCTG-1 F#ACAGCGCTCGGGACAA-1 ...
  M#CTCAACCAGTGAGTTA-1 M#GCGGAAAGTATCCCTC-1
colData names(25): DoubletEnrichment DoubletScore ... predictedGroup_Un
  predictedScore_Un

**Reply by Zepeng-Mu** [reply]
So this is a more general question then, it's not just about counting non-zero values.
But the `idx` in `rowData` should match that of `peakSet`. You could also try `rowRanges(peaks.mat)`, or use `seqnames(peaks.mat)`, `start(peaks.mat)` and `end(peaks.mat)` to get the chr, start and end of each peak.

**Reply by siyuyang1013** [reply]
actually, the number of row in seqnames(peaks.mat) does not match the number of row in peaks.mat

**Reply by Zepeng-Mu** [reply]
it's hard to say why without looking at the output
